go/shuffle/read.go

```diff
@@ -428,15 +444,25 @@ func (r *read) dequeue() message.Envelope {
 	return env
 }
 
-func (r *read) log() *log.Entry {
-	return log.WithFields(log.Fields{
-		"journal":     r.req.Shuffle.Journal,
+func (r *read) log(lvl logrus.Level, message string, extra ...interface{}) {
+	if lvl > r.logger.Level() {
+		return
+	}
+
+	var fields = logrus.Fields{
+		"build":       r.req.Shuffle.BuildId,
```

Comment:
Do you think it's worth just putting the build ID on _all_ logs in `runtime/ops.go`? I went back and forth on it, myself, but if it seems useful enough to have on all shuffle-related logs, then perhaps it's worth including on all logs? :man_shrugging:

Reply:
An interesting wrinkle is that, for example, in shuffle/coordinator.go we're logging a build that _may not be_ the same build as the task term under which the coordinator was created. It's reading under the build baked into the client's request, which is allowed to differ from the shard's.